config/config.go

```diff
@@ -446,6 +451,8 @@ type GlobalConfig struct {
 	// Keep no more than this many dropped targets per job.
 	// 0 means no limit.
 	KeepDroppedTargets uint `yaml:"keep_dropped_targets,omitempty"`
+	// Allow UTF8 Metric and Label Names
```

Comment:
(sorry, we just follow this in Prometheus; maybe we should relax this)
```suggestion
	// Allow UTF8 Metric and Label Names.
```

Reply:
Once I learn the style I'll get it :)

Reply:
We use the Go guidelines. You can find it linked somewhere here https://github.com/prometheus/prometheus/blob/main/CONTRIBUTING.md - we mostly care about proper sentences with it starting with a capital letter where appropriate and ending with a period